docs/features/effects.md

```
@@ -230,10 +247,18 @@ For 2D effects the background (secondary) color is set to black.
+
+!!! info "Image Effect"
+    * You can only have _one_ segment playing this effect
+    * Segment name must be set to the file name (like "anim1.gif") on the esp32 filesystem.
+    * Animated GIFs are _mostly_ supported natively by WLED i.e. even through a direct upload of the GIF using the file editor. 
+    * If you experience any issues you can convert the GIF using the [PixelForge Image Tool](pixelforge/image-tool).
+    * Effect is **not available on ESP8266** due to limited RAM
 
 | ID | Effect | Description | Flags | Colors | Parameters |
-|---:|:---|:---|:---:|:---:|:---|
+|:---|:---|:---|:---:|:---:|:---|
+| 53 | **Image** | Animated GIF Image. <br /> ![](../assets/images/content/effect_gifs/FX_53_comets64.gif) |  ⋮ ▦ | - | **Segment Name**:Image File Name (filename.gif) <br> **Speed:** Animation speed <br> **Blur:** Image blur <br> example image is from [marcmerlin/AnimatedGIFs](https://github.com/marcmerlin/AnimatedGIFs/blob/master/data64_2MB/gifs64/284_comets.gif)|
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)